mgmt/rest/v2/api.go

```diff
@@ -72,3 +77,21 @@ func (s *V2) BindTribeManager(tribeManager api.Tribe) {}
 func (s *V2) BindConfigManager(configManager api.Config) {
 	s.configManager = configManager
 }
+
+func Write(code int, body interface{}, w http.ResponseWriter) {
+	w.Header().Set("Content-Type", "application/json; version=2")
+	w.Header().Set("Version", "beta")
+
+	if !w.(negroni.ResponseWriter).Written() {
+		w.WriteHeader(code)
+	}
+
+	if body != nil {
+		j, err := json.MarshalIndent(body, "", "  ")
+		if err != nil {
+			restLogger.Fatalln(err)
+		}
+		j = bytes.Replace(j, []byte("\\u0026"), []byte("&"), -1)
```

Comment:
I didn't add a comment, instead I disabled HTML escape (forced us to unescape `&`).